roles/cert_manager/tasks/validate_certs.yml

```diff
@@ -20,12 +20,18 @@
     state: directory
     mode: '0755'
 
-- name: Install cert-manager cmctl CLI
-  ansible.builtin.unarchive:
-    src:
-      "https://github.com/cert-manager/cert-manager/releases/{{ cifmw_cert_manager_version }}/download/cmctl-linux-amd64.tar.gz"
-    dest: "{{ lookup('env', 'HOME') }}/bin/"
-    remote_src: true
+# jinja[invalid] is triggered by the call to the lookup_plugin.pipe
```

Comment:
Thanks! I guess it would have been better to use shell module like I had proposed initially.